### PR TITLE
Release/1.3.1

### PR DESCRIPTION
**Pull request by istreeter**

### CODE REVIEW

**Review by adatzer — APPROVED** [review]
Looks great! Resolves the issue and embedded iglu is back for `project micro` image!

Noticed that the same change was not applied for distroless image. Does that mean that distroless won't support embedded iglu?

**Comment by istreeter** [comment]
@adatzer I just pushed an update which should fix it for distroless image too.

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,7 @@
+Version 1.3.1 (2022-06-09)
```
Comment (adatzer) [inline CHANGELOG:1]: just a reminder to update the date here

**Review by adatzer — APPROVED** [review]
Embedded on distroless confirmed! LGTM!!

**Pull request merged**